src/test/Components/CreateImageWizard/steps/FirstBoot/Firstboot.test.tsx

```diff
@@ -104,6 +104,22 @@ describe('First Boot step', () => {
     await goToFirstBootStep();
     await screen.findByText('First boot configuration');
   });
+
+  // test('should validate shebang', async () => {
```

Comment:
I've written the test, but it doesn't work because as below, vitest dom doesn't work well with the editor we use.
I'll try to find a follow-up.